docs/lib/graphqlapi/mutate-data.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Create or update data
+title: Create, update, delete data
```

Comment:
This change might affect other platforms.

Reply:
updating android and flutter docs in this PR as well